docs/io-cli.md

```diff
@@ -49,8 +49,8 @@ pulsar-admin sources create options
 
 |Flag|Description|
 |----|---|
-| `-a`, `--archive` | The path to the NAR archive for the source. <br /> It also supports url-path (http/https/file [file protocol assumes that file already exists on worker host]) from which worker can download the package.
-| `--classname` | The source's class name if `archive` is file-url-path (file://).
+| `-a`, `--archive` | The absolute path to the NAR archive for the source. <br /> It also supports url-path (http/https/file [file protocol assumes that file already exists on worker host]) from which worker can download the package.
+| `--classname` | The source's class name. Mandatory if `archive` doesn't reference a NAR file.
```

Comment:
I believe the exclusion for "if `archive` doesn't reference a NAR file." only applies if the NAR file has a proper manifest entry, which may or may not be true. 

For now, I would recommend leaving it as "Mandatory" without any exclusions.

Reply:
> For now, I would recommend leaving it as "Mandatory" without any exclusions.

Ok, I'll just make it "The source's class name". Without any other additions?

Reply:
Done.